File: Module-CP/Assignments/assignment14/Pass_by_value/q8.c
#include<stdio.h>
typedef struct Distance{
    int feet;
    float inch;
}Distance;
Distance storeDate();
void display(Distance);
int main(){
    Distance d1,d2;
    d1=storeDate();
    d2=storeDate();

    display(d1);
    display(d2);
}
Distance storeDate(){
    Distance d;
    printf("Enter feet:");
    scanf("%d",&d.feet);
    printf("Enter inch:");
    scanf("%f",&d.inch);

    return d;
}
void display(Distance d){
     printf("\nfeet:%d",d.feet);
     printf("\ninch:%f",d.inch);
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/Main/q8.java
import java.util.*;
class Distance{
    int feet,inch;
}
public class q8{
    public static void main(String[] args) {
        Distance d1 = new Distance();
        Distance d2 = new Distance();

        Scanner sc = new Scanner(System.in);
        
        System.out.println("feet");
        d1.feet=sc.nextInt();
        System.out.println("inch");
        d1.inch=sc.nextInt();
        System.out.println("feet");
        d2.feet=sc.nextInt();
        System.out.println("inch");
        d2.inch=sc.nextInt();

          System.out.println("feet "+d1.feet);
          System.out.println("inch " +d1.inch);
          System.out.println("feet "+d2.feet);
          System.out.println("inch " +d2.inch);
    }
}